.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,156 @@
+    description: "Are you using PublicClientApplication,  ConfidentialClientApplication or ManagedIdentityApplication?"
+    multiple: true
+    options: 
+      - "Public"
+      - "Confidential"
+      - "Managed identity"
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: Application type
+    description: "Please select your application type"
+    multiple: true
+    options: 
+      - "Desktop"
+      - "Mobile"
+      - "Web app"
+      - "Web API"
+      - "Daemon app" 
+  validations:
+    required: true   
+
+- type: dropdown
+  attributes:
+    label: Authentication flow type
+    description: "What authentication flow was used when the issue appeared?"
+    multiple: true
+    options: 
+      - "Interactive (with an authentication broker)"
+      - "Interactive (with embedded or system browser)"
+      - "Integrated Windows Authentication"
+      - "Username and password (ROPC)"
+      - "Device code (browserless)"
+      - "Authorization code"
+      - "On-Behalf-Of"
+      - "Client credentials (service-to-service calls)"
+      - "Long-running On-Behalf-Of (OBO)"
+      - "Managed Identity"
+      - "Other"
+  validations:
+    required: true    
+
+- type: dropdown
+  attributes:
+    label: Is this a new or an existing app?
+    description: "Is this a new or existing app?"
+    multiple: false
+    options: 
+      - "The app is in production, and I have upgraded to a new version of MSAL"
+      - "The app is in production, I haven't upgraded MSAL, but started seeing this issue"
+      - "This is a new app or experiment"
+  validations:
+    required: true        
+
+- type: textarea
+  attributes:
+    label: Issue description and reproduction steps
+    description: "Briefly explain the issue you are seeing. Provide steps to reproduce or a link to a sample which demonstrates the behavior."
+  validations:
+    required: true      
+    
+- type: textarea
+  attributes:
+    label: Error message
+    description: "Provide any error messages you are receiving along with a stack trace, if possible."
+
+- type: textarea
+  attributes:
+    label: MSAL-specific logs
+    description: "Provide verbose level log messages from MSAL. [Learn more](https://learn.microsoft.com/azure/active-directory/develop/msal-logging-dotnet)"
+       
+- type: textarea
+  attributes:
+    label: Relevant code snippets
+    description: "Provide relevant code snippets that can be used to reproduce the issue."
+    render: csharp
+  validations:
+    required: false
+       
+- type: textarea
+  attributes:
+    label: Expected behavior
+    description: "Describe what you expect the behavior to be."
+  validations:
+    required: true
+    
+- type: dropdown
+  attributes:
+    label: Identity provider
+    options: 
+      - Microsoft Entra ID and/or MSA
+      - Azure B2C Basic Policy
+      - Azure B2C Custom Policy
+      - Azure Active Directory Federation Services (ADFS)
+      - Microsoft Entra External ID
```

Comment:
Is this CIAM?

Reply:
Afaik External Identities is a different thing? Unless it'll be merged into CIAM?

Reply:
Microsoft Entra External ID _is_ CIAM @gladjohn @pmaytak  😀